py-polars/Cargo.toml

```diff
@@ -101,9 +101,9 @@ name = "polars"
 crate-type = ["cdylib"]
 
 [profile.release]
-codegen-units = 1
```

Comment:
Is this intentional or leftover from testing?

Reply:
same for other changes in this file.

Reply:
Oh thanks. leftover indeed.